sxg_rs/src/signature/js_signer.rs

```diff
@@ -73,9 +74,9 @@ impl Signer for JsSigner {
     }
 }
 
-fn raw_sig_to_asn1(raw: Vec<u8>) -> Result<Vec<u8>, String> {
+fn raw_sig_to_asn1(raw: Vec<u8>) -> Result<Vec<u8>> {
     if raw.len() != 64 {
-        return Err(format!("Expecting signature length to be 64, found {}.", raw.len()));
+        return Err(Error::msg(format!("Expecting signature length to be 64, found {}.", raw.len())));
```

Comment:
Should the 64 be declared a constant somewhere? and then passed into this error message construct?

Reply:
Done.